pkg/acsengine/engine.go

```diff
@@ -635,7 +635,8 @@ func getPackageGUID(orchestratorType api.OrchestratorType, masterCount int) stri
 func getDCOSCustomDataPublicIPStr(orchestratorType api.OrchestratorType, masterCount int) string {
 	if orchestratorType == api.DCOS173 ||
 		orchestratorType == api.DCOS184 ||
-		orchestratorType == api.DCOS187 {
+		orchestratorType == api.DCOS187 ||
+		orchestratorType == api.DCOS188 {
```

Comment:
I was trying to find dcoscustomdata188.t but couldn't for some reason. I refreshed, now I see it.